03-task-manager/starter/app.js

```diff
@@ -1,22 +1,24 @@
-console.log("Task Manager App");
+console.log("Task Manager App Starting");
 
 const express = require(`express`);
 const app = express();
 const tasks = require("./routes/tasks");
 const connectDB = require(`./db/connect`);
+const notFound = require("./middleware/not-found");
+const errorHandlerMiddleware = require("./middleware/error-handler");
 require(`dotenv`).config();
 
 //middleware
+app.use(express.static(`./public`));
 app.use(express.json());
 
 //routes
-app.get("/hello", (req, res) => {
-  res.send("Task Manager App");
-});
 
 app.use(`/api/v1/tasks`, tasks);
+app.use(notFound);
```

Comment:
Not a big deal either way but usually organization convention puts imports above any routes